### ISSUE TITLE
Which XCLIP pretraining to use?

### ISSUE THREAD

**Issue by versteckterRabe** [chenhaoxing/DeMamba#13]
Hi thank you very much for your work. I was wondering, in order to train this model which pretrained weights of XCLIP should we use? The code points to some local weights which are not available on the modelscope link

**Comment by chenhaoxing** [comment]
https://huggingface.co/microsoft/xclip-base-patch16

**Comment by versteckterRabe** [comment]
Hi, thank you very much for your reply. Let me ask a follow up question.  What data should one use as a a baseline to retrain and compare? As a training set it should use all the models in the table from Zeroscope to Opensora, but what files should be used as validation set? All those in the GenVideo-Val.zip? Or are those used only for test and if so what data should be used as validation? Also would it be possible to have the code to test a trained netwowk?
Thank you
